types/office-js-preview/index.d.ts

```diff
@@ -15314,6 +15531,38 @@ declare namespace Office {
          *                   OR a {@link Office.ReplyFormData | ReplyFormData} object that contains body or attachment data and a callback function.
          */
         displayReplyAllForm(formData: string | ReplyFormData): void;
+        /**
+         * Displays a reply form that includes the sender and all recipients of the selected message or the organizer and all attendees of the
+         * selected appointment.
+         *
+         * In Outlook on the web, the reply form is displayed as a pop-out form in the 3-column view and a pop-up form in the 2- or 1-column view.
+         *
+         * If any of the string parameters exceed their limits, `displayReplyAllFormAsync` throws an exception.
+         *
+         * When attachments are specified in the `formData.attachments` parameter, Outlook on the web and desktop clients attempt to download
+         * all attachments and attach them to the reply form. If any attachments fail to be added, an error is shown in the form UI.
+         * If this isn't possible, then no error message is thrown.
```

Comment:
Am I reading this right that if no error can be displayed in the form, then no error is thrown?

Reply:
Yes, it seems that way.